src/test/modelling/test_modelling.cxx

```diff
@@ -178,18 +178,21 @@ void modellingTests::run_tests()
       std::cerr << "\nTesting the creation of Model Matrix based on Plasma Data..." << std::endl;     
       PatlakPlot patlak_plot;      
       const unsigned int starting_frame=23;
-      ModelMatrix<2> stir_model_matrix=(patlak_plot.get_model_matrix(sample_plasma_data_in_frames,time_frame_def,starting_frame));
+      patlak_plot._plasma_frame_data=sample_plasma_data_in_frames;
+      patlak_plot._frame_defs=time_frame_def;
+      patlak_plot._starting_frame=starting_frame;
+      patlak_plot.set_up();
+      ModelMatrix<2> stir_model_matrix=(patlak_plot.get_model_matrix());
       ModelMatrix<2> mathematica_model_matrix;
       mathematica_model_matrix.read_from_file(this->add_directory("math_model_matrix.in"));
-      stir_model_matrix.uncalibrate(10);
```

Comment:
The current `create_model_matrix()` does the uncalibration inside. `ModelMatrix` only allows you to do this once, thus `stir_model_matrix` can not be uncalibrated. If we leave this line here, the tests in 194 failed with the division by 10. 

Hopefully this does not change function, as `ModelMatrix.uncalibrate` is being tested here anyway, just inside `patlak_plot.set_up()` in the call to `patlak_plot_create_model_matrix()`

Reply:
I can see we don't want to call `uncalibrate` here, but I cannot see why we'd get some results. `create_model_matrix` uses `his->_cal_factor`, but that should default to 1, not 10. Wouldn't you have to set it above (before `set_up()`)?

Reply:
ah sorry. I see you removed the `/10.F.` below. ok, test should work. But I suggest to set the `_cal_factor` anyway, and use the value of `cal_factor as opposed to a hard-wired `10.F` below.

Reply:
@KrisThielemans done! Test pass in my machine, well see how CI goes.